src/main/java/com/ioet/bpm/skills/domain/Category.java

```diff
@@ -0,0 +1,19 @@
+package com.ioet.bpm.skills.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import javax.validation.constraints.NotBlank;
+
+@Data
+@AllArgsConstructor
+public class Category {
```

Comment:
Is there a reason to don't have this field marked as private?

Reply:
Done